platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/emergency/EmergencyReconnectLearner.java

```diff
@@ -36,29 +40,38 @@ public class EmergencyReconnectLearner {
     private final EmergencySignedStateValidator validator;
 
     /**
-     * @param emergencyRecoveryFile
-     * 		the emergency recovery file used for this reconnect
-     * @param reconnectController
-     * 		controls reconnecting as a learner
+     * Used to submit status actions
+     */
+    private final StatusActionSubmitter statusActionSubmitter;
+
+    /**
+     * @param emergencyRecoveryFile the emergency recovery file used for this reconnect
+     * @param reconnectController   controls reconnecting as a learner
+     * @param statusActionSubmitter used to submit status actions
      */
     public EmergencyReconnectLearner(
-            final EmergencyRecoveryFile emergencyRecoveryFile, final ReconnectController reconnectController) {
+            final EmergencyRecoveryFile emergencyRecoveryFile,
```

Comment:
`@NonNull` or `@Nullable` markings on the other arguments.

Reply:
added